test/core/end2end/tests/max_connection_idle.cc

```diff
@@ -69,8 +87,9 @@ CORE_END2END_TEST(RetryHttp2Test, MaxConnectionIdle) {
           .Set(GRPC_ARG_INITIAL_RECONNECT_BACKOFF_MS,
                Duration::Seconds(1).millis())
           .Set(GRPC_ARG_MAX_RECONNECT_BACKOFF_MS, Duration::Seconds(1).millis())
-          .Set(GRPC_ARG_MIN_RECONNECT_BACKOFF_MS,
-               Duration::Seconds(5).millis()));
+          .Set(GRPC_ARG_MIN_RECONNECT_BACKOFF_MS, Duration::Seconds(5).millis())
+          // Avoid transparent retries for this test.
+          .Set(GRPC_ARG_ENABLE_RETRIES, false));
```

Comment:
If we want to disable retries, should we maybe just run this test in the NoRetry fixture?

Reply:
I think this is the right suite, but that it's misnamed slightly.